src/java/org/apache/cassandra/config/AccordSpec.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.config;
+
+public class AccordSpec
+{
+    public volatile boolean enabled = false;
+
+    public volatile String journal_directory;
+
+    public volatile OptionaldPositiveInt shard_count = OptionaldPositiveInt.UNDEFINED;
+
+    public volatile DurationSpec.IntSecondsBound progress_log_schedule_delay = new DurationSpec.IntSecondsBound(1);
```

Comment:
unless we plan to have JMX/VTable overrides of these properties, we should not add `volatile`... For example `enabled` can not change during the life of the JVM as we only add the accord tables on boot...  same for most of these configs, they are init only

Reply:
I was primarily thinking about tests